server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/sql/TestSqlAllDescribeStatements.java

```diff
@@ -47,22 +47,27 @@ class TestSqlAllDescribeStatements {
           + "           EXEC SQL\n";
 
   private static final String DESCRIBE_CURSOR =
-      TEXT + "            DESCRIBE CURSOR C1 INTO SQLDA\n" + "           END-EXEC.";
+//      TEXT + "            DESCRIBE CURSOR C1 INTO SQLDA\n" + "           END-EXEC.";
+  TEXT + "            DESCRIBE CURSOR C1 INTO :{$HOSTVAR}\n" + "           END-EXEC.";
 
   private static final String DESCRIBE_INPUT =
-      TEXT + "            DESCRIBE INPUT STMT1_NAME INTO SQLDA;\n" + "           END-EXEC.";
+//      TEXT + "            DESCRIBE INPUT STMT1_NAME INTO SQLDA;\n" + "           END-EXEC.";
```

Comment:
Will it make sense to have separate test for this cases?

Reply:
updated. Removed the commented code (for which compilation fails) as discussed.